torch/_inductor/compile_fx_ext.py

```diff
@@ -481,12 +481,12 @@ def serialize_compile(
                 fake_tensor_mode,
             ).serialize()
             return (input, constants)
-        except (AttributeError, BypassFxGraphCache):
+        except (AttributeError, BypassFxGraphCache) as e:
             # For example: AttributeError: Can't pickle local object
             # 'make_opaque_unary_fn.<locals>.OpaqueUnaryFn'
 
             # TODO: scuba record about not being able to do this?
-            log.debug("Unable to pickle input graph or example inputs", exc_info=True)
+            log.debug("Unable to pickle input graph or example inputs: %s" % e, exc_info=True)
```

Comment:
Why exc_info=True insufficient?

Reply:
Oh, you're correct. I made a mistake. Now I tried to make it a warning. I feel it worth a warning if we made something meant to be serialized.